Assets/.keep

```diff
@@ -0,0 +1 @@
+Workaround for CloudBuild automation

```

Comment:
Yes, go ahead and remove. We could always have a separate project for CloudBuild purposes that uses a submodule for EditorVR.